.github/workflows/docker-images.yml

```diff
@@ -55,6 +55,7 @@ jobs:
               run: |
                   pushd ./scripts/actions/docker-images
                   bun run-ci
+                  yarn format scripts/actions/docker-images
```

Comment:
I don't see how this will work, considering the action doesn't call setup-node or yarn at all... Also I'd rather have this file ignored from formatting, the only reason it's even indented is so we can easily see in diff's what changed

Reply:
Ah, right, that's why you run bun on the previous line. Sure, having biome ignore that file is fine with me as well.